Dockerfile.uiforge

```
@@ -0,0 +1,41 @@
+
+RUN npm ci --ignore-scripts --legacy-peer-deps && \
+    npm rebuild @resvg/resvg-js && \
+    npm run build
+
+# Copy font assets into dist for satori (import.meta.url relative path)
+RUN cp -r src/assets dist/assets
+
+# ── Stage 2: Runtime with mcpgateway.translate ───────────────
+FROM python:3.12-slim
+
+RUN apt-get update -qq && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)